doc-tests/README.md

```diff
@@ -0,0 +1,134 @@
+# Testable Deployment Documentation
+This directory contains the code that is used to daily integrate all potential changes in the
+microservices and especially in the descriptions of how these are deployed on top of different
+suported targets.
+
+This implies that the user-facing documentation is the canonical source for our tests, preventing
+out-of-date documentation.
+
+
+## Phases
+The test runner has four phases:
+
+- `pre-install`, in which required additonal software should be installed, such as Terraform.
+  If this phase fails, the tests just aborts, without attempting to destroy the infrastructure.
+  *NOTE*: This implies that it is very important to also install any the tool that is required
+  to tear down the infrastructure.
+- `create-infrastructure`, in which the cloud resources are created. If this phase fails,
+  the tests are skipped, but the `destroy-infrastructure` phase is executed.
+- `run-tests`, which should contain instructions that validate that the infrastructure is up and
+   running and that the microservices are deployed correctly.
+-  `destroy-infrastructure`, which must tear down the infrastructure.
+
+### Secrets phase
+There is an additional `require-env` phase, that can be used to pass in environmental variables to
+the steps. If they are not present, testing the documentation will result in an error message that
+these variables are not present.
+
+## Defining phases and steps
+Multiple shell snippets (steps) can be added to a phase.
+The phases are run in the order give in the previous section, after which each snippet is executed
+in the order of declaration in the file.
+
+The following syntaxes are supported to add snippets:
+
+ 1. Single line annotation, not visibile in Markdown output
+
+    &lt;!-- deploy-test PHASE [VALUE]* --&gt;
+
+ 2. Hidden multiline annotation
+
+    &lt;!-- deploy-test-start PHASE [VALUE]*
```

Comment:
no, it's the html comment that has not ended yet.

two  online comments

    <!-- one line comment 1 -->
    CONTENT A
    <!-- one line comment 2-->

vs 

    <!-- several  
    oneline
    CONTENT B
    comments
    -->

Therefore  `CONTENTA` will be shown in the HTML, the and `CONTENTB` will not.